docs/astro/src/content/docs/reference/std-widgets/views/textedit.mdx

```diff
@@ -98,8 +98,9 @@ TextEdit {
 ```
 
 ### key-pressed(KeyEvent) -> EventResult
-Invoked when a key is pressed, the argument is a <Link type="KeyEvent" /> struct. The returned `EventResult` 
-indicates whether to accept or ignore the event. Ignored events are forwarded to the parent element.
+Invoked when a key is pressed, the argument is a <Link type="KeyEvent" /> struct. Use this callback to
+handle keys before `TextEdit` does. Return `accept` to indicate that you've handled the event, or return
+`reject` to let `TextEdit` handle it.
 
 ### key-released(KeyEvent) -> EventResult
```

Comment:
This one is still wrong, as well as the `key-released` callback for `LineEdit` itself.